spec/factories/restaurant_factory.rb

```diff
@@ -23,7 +23,7 @@
     cuisine { Restaurant::E_CUISINES.sample }
     accept10bis { [true, false].sample }
     rating { Faker::Number.within(0.0..3.0) }
-    max_delivery_time_min { [1..720].sample }
+    max_delivery_time_min { Faker::Number.within(1..720) }
```

Comment:
12 hours until getting my food?? 😨

Reply:
happened to me once... (true story ;)... 
either way added MAX_DELIVERY_TIME constant and set it to 5 hrs...